iib/workers/tasks/build.py

```diff
@@ -755,14 +752,24 @@ def handle_add_request(
                     ' '.join(excluded_bundles),
                 )
 
-        _opm_index_add(
-            temp_dir,
-            resolved_bundles,
-            prebuild_info['binary_image_resolved'],
-            from_index_resolved,
-            overwrite_from_index_token,
-            (prebuild_info['distribution_scope'] in ['dev', 'stage']),
-        )
+        if is_fbc:
+            opm_registry_add_fbc(
+                base_dir=temp_dir,
+                bundles=resolved_bundles,
+                binary_image=prebuild_info['binary_image_resolved'],
+                from_index=from_index_resolved,
+                overwrite_from_index_token=overwrite_from_index_token,
+                overwrite_csv=(prebuild_info['distribution_scope'] in ['dev', 'stage']),
+            )
+        else:
+            _opm_index_add(
+                temp_dir,
```

Comment:
Could you use the same mapping as in opm_registry_add_fbc(value=value)?

Reply:
sure